### ISSUE TITLE
git config

### ISSUE THREAD

**Issue by data-if** [lk-geimfari/macbook#6]
```sh
git config --global user.name "${USER_EMAIL}"
```

may be...

```sh
git config --global user.name "${USERNAME}"
```

... ❓

**Comment by data-if** [comment]
try to push fix, but:

```sh
remote: Permission to lk-geimfari/macbook.git denied

fatal: unable to access 'https://github.com/lk-geimfari/macbook.git/': The requested URL returned error: 403
```

**Comment by lk-geimfari** [comment]
Fixed in [6b413c1](https://github.com/lk-geimfari/macbook/commit/6b413c1cba883fe802112ab6a8cd65ede3942078)
